### PR TITLE
Parse WMS and WFS layers from URL parameters

### PR DESCRIPTION
**Pull request by raitisbe**
port of https://github.com/hslayers/hslayers-ng/pull/1311

### CODE REVIEW

File: projects/hslayers/src/components/add-layers/add-layers.component.ts [hunk 1]
```diff
@@ -1,4 +1,6 @@
 import {Component} from '@angular/core';
+import {HsAddLayersWfsService} from './wfs/add-layers-wfs.service';
```
Comment (jmacura) [inline projects/hslayers/src/components/add-layers/add-layers.component.ts:2]: You can delete these imports. Looks like I forgot it during clean-up.

File: projects/hslayers/src/components/add-layers/add-layers.component.ts [hunk 2]
```diff
@@ -13,16 +15,17 @@ import {HsShareUrlService} from '../permalink/share-url.service';
 export class HsAddLayersComponent {
   showDetails: boolean;
   type: string;
-  types;
+  types: any[];
 
   constructor(
+    public hsAddLayersWfsService: HsAddLayersWfsService,
```
Comment (jmacura) [inline projects/hslayers/src/components/add-layers/add-layers.component.ts:21]: You can delete these injections. Looks like I forgot it during clean-up.

File: projects/hslayers/src/components/add-layers/add-layers.component.ts [hunk 3]
```diff
@@ -73,13 +76,13 @@ export class HsAddLayersComponent {
...
   /**
-   * Change detail panel template according to selected type
-   *
    * @function templateByType
-   * @returns {string} template Path to correct type template
+   * @description Change detail panel template according to selected type
+   * @todo unused
+   * @returns {string} Path to correct type template
    */
-  templateByType(): string {
```
Comment (jmacura) [inline projects/hslayers/src/components/add-layers/add-layers.component.ts:81]: This method is no longer used. I intended to delete it, but obviously forgot it in the end.

**Review by jmacura — APPROVED** [review]

**Comment by FilipLeitner** [comment]
[Load composition at startup from url param](https://trello.com/c/lFbCQZXq/123-load-composition-at-startup-from-url-param)

**Comment by FilipLeitner** [comment]
[Load composition at startup from url param](https://trello.com/c/lFbCQZXq/123-load-composition-at-startup-from-url-param)

**Pull request merged**